Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Lich King's Mind Control

## Changelog:
- Lich King's vassals are less likely to revolt if has high Dark Authority and more likely if it has low.
- Having high Dark Authority, undead vassals are less likely to refuse Lich King's interactions.


## Tests:
- [x] Revert tests


# How to test:
Check these features. Remember, you can lower and raise Dark Frenzy with the Debug Mode enabled by clicking the Lich King and looking under the **_DEBUG Interactions_**.

So you may raise/low Dark Frenzy and see how the AI reacts.
